packages/react/src/components/accordion/accordion.stories.tsx

```diff
@@ -39,6 +39,7 @@ const {
       compactBlackSections: basicSections,
       neutralSections: basicSections,
       subSections: subSections(templates),
+      renvooiSections: subSections(templates),
```

Comment:
Waarom hier niet de renvooiSections? Mss moeten we deze functionaliteit in zijn geheel niet willen etaleren, omdat de React afnemers er toch niets mee doen.

Reply:
Ah. Ik snap hem. We etaleren idd de story `RenvooiSections` niet in de React Stoprybook. ✅  Moeten uiteraard wel een `storyTemplates`-function retourneren die een `renvooiSections` retourneert.